addon/utils/pretty-percentage.js

```diff
@@ -0,0 +1,10 @@
+export default function prettyPercentage(decimalValue, precision=0) {
```

Comment:
One more thought... Does this functionality belong in this package? Right now this package simply does unit conversions. Perhaps it's wise to have number formatting utils in an additional package?